_includes/package_readme.html

```diff
@@ -14,6 +14,6 @@
     | split: "<br />"
```

Comment:
I wanted to convert newlines to tags to be able to split them and remove n lines from the header.

Split by `\n` does not work but would solve the problem.